modules/inflowwind/python-lib/inflowwind_library.py

```diff
@@ -0,0 +1,206 @@
+#**********************************************************************************************************************************
+# LICENSING
+# Copyright (C) 2021 National Renewable Energy Laboratory
+#
+# This file is part of InflowWind.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#**********************************************************************************************************************************
+#
+# This is the Python-C interface library for InflowWind
+# Usage: THIS LIBRARY IS NOT TO BE CHANGED OR EDITED BY THE USER
+from ctypes import (
+	CDLL,
+    POINTER,
+    create_string_buffer,
+    byref,
+    c_byte,
+    c_int,
+    c_double,
+    c_float, 
+    c_char,
+    c_char_p, 
+    c_wchar, 
+    c_wchar_p,
+    c_bool
+)
+import numpy as np
+
+class InflowWindLibAPI(CDLL):
+    # Human readable error levels from IfW.
+    error_levels = {
+        0: "None",
+        1: "Info",
+        2: "Warning",
+        3: "Severe Error",
+        4: "Fatal Error"
+    }
+
+    def __init__(self, library_path):
+        super().__init__(library_path)
+        self.library_path = library_path
+
+        self._initialize_routines()
+        self.ended = False                  # For error handling at end
+
+        # Create buffers for class data
+        self.abort_error_level = c_int(4)
+        self.error_status = c_int(0)
+        self.error_message = create_string_buffer(1025)
+
+        # This buffer for the channel names and units is set arbitrarily large
+        # to start.  InflowWind only has a maximum of 9 outputs at present, but
+        # may be expanded.  Channel name and unit lengths are currently hard
+        # coded to 20 (this must match ChanLen in NWTC_Base.f90).
+        self._channel_names = create_string_buffer(20 * 4000)
+        self._channel_units = create_string_buffer(20 * 4000)
+
+        self.dt = c_double(0)               # InflowWind must be passed
```

Comment:
Is this correct?  If we don't set `dt` in the driver that calls this, an error of `must be real number, not c_double` when `dt` is passed on the call it `inflowwind_init_c`.